dev/com.ibm.ws.logging.core/src/com/ibm/websphere/ras/TrConfigurator.java

```diff
@@ -236,8 +236,11 @@ public TrService call() throws Exception {
      *            the {@link TraceComponent} that was registered
      */
     static void traceComponentRegistered(TraceComponent tc) {
-        for (TraceComponentChangeListener listener : registeredListeners.get()) {
-            listener.traceComponentRegistered(tc);
+        Set<TraceComponentChangeListener> listeners = registeredListeners.get();
+        if (listeners.size() > 0) {
```

Comment:
Should this use `!listeners.isEmpty()`

Reply:
I can change it to that.  It is the same effect though.  I should have possibly changed it to != 0 instead of > 0.